app/models/miq_server/configuration_management.rb

```diff
@@ -38,9 +38,7 @@ def config_activated(data)
   end
 
   def sync_config
-    sync_worker_monitor_settings
-    sync_child_worker_settings
-    $log.log_hashes(@worker_monitor_settings)
+    worker_management.sync_config
```

Comment:
heh. yes I know that I said delegates may be early but I keep seeing this and thinking a delegate would be nice here.

Reply:
:+1: done